network.koesling.shm-modbus.yml

```diff
@@ -0,0 +1,128 @@
+id: network.koesling.shm-modbus
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: shm-modbus
+finish-args:
+  - --device=shm
```

Comment:
Required because the purpose of the application is to make its data available via a shared memory.

Reply:
I'm not so sure, `shm` is a relic from a misguided attempt to allow JACK to work.

Also, this doesn't display anything?

Reply:
shm is required. The purpose of this application is to share some of its data with other applications. If the applications do not have a common shared memory, the application is useless. 

It doesn't display anything, it is a command line based application.